src/chat/components/InputToolbar.tsx

```diff
@@ -44,50 +23,62 @@ interface InputToolbarProps {
 export default function InputToolbar({ onSubmit }: InputToolbarProps) {
   const classes = useStyles();
   const intl = useIntl();
-  const [message, setMessage] = React.useState('');
 
-  const handleChange = (e: React.ChangeEvent<HTMLTextAreaElement>) => {
-    setMessage(e.target.value);
+  const [value, setValue] = useState(0);
+
+  const handleTabChange = (_event: React.ChangeEvent<{}>, newValue: number) => {
```

Comment:
Curious - why did you put an underscore in the argument name `_event`? Is this a pattern we're following?

Reply:
`_` is typically used as a placeholder variable. 
 I wanted to indicate an unused parameter 🙂